app/Http/Controllers/HubController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class HubController extends Controller
+{
+    /**
+     * Returns the landing page of the conference hub
+     */
+    public function dashboard()
+    {
+        return view('myhub.home');
+    }
+
+    /**
+     * get view for profile information
+     */
+    public function getProfileInfo()
```

Comment:
Maybe just rename the function to `profile`? This way it would allign with `dashboard` naming for instance.

Reply:
I think I'll even remove it for now (it was a snipped from the old edition), thanks